docs/docs/installation/pypi.mdx

```diff
@@ -129,21 +121,33 @@ First, start by installing `apache-superset`:
 pip install apache-superset
 ```
 
-Then, you need to initialize the database:
+Superset configurations are stored in a file.  One of these configurations, a `SECRET_KEY`, is required for the application to start. Create your config file:
+```
+touch superset/superset_config.py
+```
+And make this file findable by adding its path as an environment variable:
+```
+export SUPERSET_CONFIG_PATH=superset/superset_config.py
```

Comment:
This is temporary, this env needs to persist. I have added it .zshrc (for macOS)

Reply:
Good suggestion, I've implemented with a link to StackOverflow explaining how to do this.